cmd/main.go

```diff
@@ -58,11 +63,9 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
```

Comment:
Please don't, let's make sure this variables only available in main function for now.

Reply:
ok ,i revert this soon.